code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -779,3 +779,188 @@ SYNDICATE BLACK OPS
 	specflags = list(RADIMMUNE,VIRUSIMMUNE,NOBLOOD,PIERCEIMMUNE,EYECOLOR)
 	sexes = 0
 
+
+
+/datum/species/pony
+	id = "pony"
+	name = "Pony"
+	roundstart = 1
+
+	sexes = 1
+	exotic_blood = /datum/reagent/friendship
+
+	no_equip = list(slot_shoes, slot_gloves, slot_w_uniform)
+	// slots the race can't equip stuff to
+
+	nojumpsuit = 1
+
+	default_features = list("wings" = "None", "alicorn" = "None")
+	mutant_bodyparts = list("wings", "alicorn")
+
+	punchdamagelow = 1
+	punchdamagehigh = 5
+	punchstunthreshold = 5
+	// Weaker than humans
+
+	// species flags. these can be found in flags.dm
+	specflags = list(MUTCOLORS, HAIR, EYECOLOR, NOBLOOD, PIERCEIMMUNE)
+
+	attack_verb = "kick"
+
+	var/pony_hair = null
+	var/pony_tail = null
+	var/alicorn_color = null
+	var/list/pony_types = list(
+		"rarity", "fleur", "twilight",
+		"pinkie", "lyra", "vinyl",
+		"whooves", "rainbow", "fluttershy")
+
+/datum/species/pony/New()
+	if(default_features["alicorn"] == "None" && prob(15))
```

Comment:
This shouldnt randomly set itself. It fucks with the character setup screen image.